package.json

```diff
@@ -0,0 +1,19 @@
+
+{
+
```

Comment:
where is the rest of your json package?  This repo does not even function as you cannot lode node without the proper json.  This is what it SHOULD look like
{
  "name": "tdd-strict-equal",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",  ****IN YOUR CASE strictequal.js****
  "scripts": {
    "lint": "./node_modules/.bin/eslint --format codeframe .",
    "lint:fix": "./node_modules/.bin/eslint --fix --format codeframe .",
    "test": "./node_modules/.bin/mocha tests/*.test.js" ****THIS WILL RUN ALL TESTS IN YOU TESTS FILE****
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "chai": "^4.2.0",
    "eslint": "^6.8.0",
    "mocha": "^7.1.2"
  }
}